lib/Cake/Cache/Cache.php

```diff
@@ -578,4 +578,41 @@ public static function remember($key, $callable, $config = 'default') {
 		return $results;
 	}
 
+/**
+ * Write data for key into a cache engine if it doesn't exist already.
+ *
+ * ### Usage:
+ *
+ * Writing to the active cache config:
+ *
+ * `Cache::add('cached_data', $data);`
+ *
+ * Writing to a specific cache config:
+ *
+ * `Cache::add('cached_data', $data, 'long_term');`
+ *
+ * @param string $key Identifier for the data
```

Comment:
We did have a sniff in the new coding standard to ensure period at the end. But it was disabled since no one has had the time to update existing docblocks as required :)